src/ocean/util/log/Config.d

```diff
@@ -633,8 +662,8 @@ public void configureLogger
 
     if (console_enabled)
     {
-        Layout console_log_layout = (settings.console_layout.length)
-            ? newLayout(settings.console_layout)
+       Layout console_log_layout = (settings.console_layout.length)
```

Comment:
> mmm...I don't think so. You might need to add one space before `Layout`

One level of indent = 4 spaces. There were 5 before. I do not do this manually, my editor caught it.